### PR TITLE
[Event] move report event link

### PR DESCRIPTION
**Pull request by JeremieSamson**
Deplacer "Signaler cet événement" et le mettre en bleu clair

### CODE REVIEW

**Comment by thomasbisignani** [comment]
:1st_place_medal:

File: front/style/pages/_committee.scss [hunk 1]
```diff
@@ -471,6 +471,10 @@
         }
     }
 
+    .box-right {
```
Comment (ottaviano) [inline front/style/pages/_committee.scss]: can we add this general rule to `_layout.scss` file with `float-right` as the name?

Comment (JeremieSamson) [reply front/style/pages/_committee.scss]: We had the same thought but @thomasbisignani told me there is a pull-righ / pull-left from font awesome css we should use as it already exists (next commit).

Comment (ottaviano) [reply front/style/pages/_committee.scss]: hm, do we use FA?

Comment (JeremieSamson) [reply front/style/pages/_committee.scss]: yes sir => https://github.com/EnMarche/en-marche.fr/blob/master/templates/base.html.twig#L97

Comment (ottaviano) [reply front/style/pages/_committee.scss]: lol, thanks, I don't find it in package.json

**Pull request merged**